tensorflow_probability/python/stats/__init__.py

```diff
@@ -40,7 +40,13 @@
 from tensorflow_probability.python.stats.sample_stats import cholesky_covariance
 from tensorflow_probability.python.stats.sample_stats import correlation
 from tensorflow_probability.python.stats.sample_stats import covariance
+from tensorflow_probability.python.stats.sample_stats import finalize_running_covariance
```

Comment:
Let's also move these functions into the experimental directory, and either make them private or inline them into methods.  One goal of making a class or tuple to hold the functions was to reduce the number of symbols in the stats namespace.

Reply:
Sounds good! So to be clear, everything should live in experimental?

Reply:
Yes, I think that's the right place to start with this one.